contrail/functions

```diff
@@ -1,6 +1,19 @@
 
 with_venv=tools/with_venv.sh
 
+declare -A sku_dict
+sku_dict['2014.1']='icehouse'
+sku_dict['2014.2']='juno'
+sku_dict['2015.1.2']='kilo'
+sku_dict['12.0']='liberty'
+sku_dict['13.0']='mitaka'
+
+function get_sku() {
+    ver=`${with_venv} nova-manage version`
```

Comment:
BTW, i dont see the upstream changesets. I think this pull request is just the merge conflict resolution. Also the merge conflict resolution for test-requirements.txt is missing and run-contrail-tempest.sh is not complete.